ui/app/AppLayouts/Communities/panels/OverviewSettingsChart.qml

```diff
@@ -0,0 +1,371 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml 2.15
+
+import StatusQ.Popups 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+import utils 1.0
+
+StatusChartPanel {
+    id: root
+    /**
+        *  Flat model to use for the chart containing timestamps
+        *  type: {Array}
+    */
+    property var model: []
+
+    QtObject {
+        id: d
```

Comment:
In general, some of these `d` properties / functions have the potential of being part of a `chart-helper` file. WDYT?

Reply:
Yes, I agree. Parts of this code will be reused for the other chart types (at least in overview settings) and I was thinking it would make sense to split this code into reusable parts once we get there. As a result, I was thinking to leave it for the next phase when we know for sure what's specific to this chart and what is reusable. Might as well result in new generic components in StatusQ.